dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/fat/AcmeSimpleTest.java

```diff
@@ -36,6 +36,7 @@
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.junit.After;
 import org.junit.AfterClass;
+import org.junit.Assume;
```

Comment:
Doesn't appear you use this.

```suggestion
```

Reply:
That is used for Assume.assumeTrue in the Windows check.